sha1-validation/test.json

```diff
@@ -7,6 +7,7 @@
     "type": "bash",
     "cleanup": true,
     "skipWhen": [
+      "non-packaged", // skip during cross-bld-sdk/tarball validation
```

Comment:
This test is checking if we're applying a certain patch for RHEL9.
I wonder why this test does not fail on hydra's vmr builds (since they don't include the patch) :thinking: 
We should take a closer look.

cc @nicrowe00